torch/onnx/_internal/exporter/_compat.py

```diff
@@ -72,6 +74,9 @@ def _from_dynamic_axes_to_dynamic_shapes(
     for input_name, axes in dynamic_axes.items():
         # input_name can be either from inptu_names or from the model inputs
         if input_name not in input_names_to_model_inputs:
+            if input_name in output_names:
+                # User specified an output name as a dynamic axis, so we skip it
+                continue
```

Comment:
```suggestion
    for input_name, axes in dynamic_axes.items():
        if input_name in output_names:
            # User specified an output name as a dynamic axis, so we skip it
            continue
        # input_name can be either from inptu_names or from the model inputs
        if input_name not in input_names_to_model_inputs:
```

this can be moved up

Reply:
nit: inptu_names typo

Reply:
Done